src/libs/actions/IOU.js

```diff
@@ -465,13 +465,16 @@ function getMoneyRequestInformation(
         billable,
     );
 
-    const uniquePolicyRecentlyUsedCategories = allRecentlyUsedCategories
-        ? _.filter(
-              allRecentlyUsedCategories[`${ONYXKEYS.COLLECTION.POLICY_RECENTLY_USED_CATEGORIES}${iouReport.policyID}`],
-              (recentlyUsedPolicyCategory) => recentlyUsedPolicyCategory !== category,
-          )
-        : [];
-    const optimisticPolicyRecentlyUsedCategories = [category, ...uniquePolicyRecentlyUsedCategories];
+    let optimisticPolicyRecentlyUsedCategories;
+    if (category) {
```

Comment:
This condition is out of an initial scope, but I've found a problem that an empty string could be set to recently used categories.

Reply:
@hoangzinh, I've just created an issue for this fix - https://github.com/Expensify/App/issues/28470. I will omit these lines soon.

Reply:
Reverted - 43144a92d07b5d9ad50917f36ffaa3ce349b4f0b.